docs/rfcs/0000-rfc-template.md

```diff
@@ -0,0 +1,126 @@
+
+The section should return to the examples given in the guide level explanation below, and explain more fully how the detailed proposal makes those examples work.
+
+### Values
+
+Describe what values does the proposed feature reflect. See RFC-0001[FIXME: Add link to RFC1].
+
+### User stories
+
+Explain what is the use case of the proposed feature and how it would benefit the user.
+
+### Guide-level explanation
+
+Explain the proposal as if it was already a feature of the project and this would be the documentation for that feature.
+
+- Introducing new named concepts.
+- Explaining the feature largely in terms of examples.
+- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
+- If applicable, describe the differences between teaching this to a Racklet administrator versus a Racklet end user.
+
+### Risks and Mitigations
+
+What are the risks of this proposal and how do we mitigate.
+Think broadly.
+For example, consider both security and how this will impact the larger kubernetes ecosystem.
+
+## Drawbacks
+
+Why should we *not* do this?
+
+## Rationale and alternatives
+
+- Why is this design the best in the space of possible designs?
+- What other designs have been considered and what is the rationale for not choosing them?
+- What is the impact of not doing this?
+
+## Prior art
+
+Discuss prior art, both the good and the bad, in relation to this proposal.
+A few examples of what this can include are:
+
+- For community proposals: Is this done by some other community and what were their experiences with it?
+- For other teams: What lessons can we learn from what other communities have done here?
+- Papers: Are there any published papers or great posts that discuss this? If you have some relevant papers to refer to, this can serve as a more detailed theoretical background.
+
+This section is intended to encourage you as an author to think about the lessons from other projects, provide readers of your RFC with a fuller picture.
+If there is no prior art, that is fine - your ideas are interesting to us whether they are brand new or if it is an adaptation from other projects.
+
+## Unresolved questions
+
+- What parts of the design do you expect to resolve through the RFC process before this gets merged?
+- What parts of the design do you expect to resolve through the implementation of this feature before stabilization?
+- What related issues do you consider out of scope for this RFC that could be addressed in the future independently of the solution that comes out of this RFC?
+
+## Future possibilities
+
+Think about what the natural extension and evolution of your proposal would
+be and how it would affect the project as a whole in a holistic
+way. Try to use this section as a tool to more fully consider all possible
+interactions with the project in your proposal.
+Also consider how this all fits into the roadmap for the project
+and of the relevant sub-team.
```

Comment:
We don't really have sub-teams right now, I would remove this for now and update the template later if (when) we get there.

Reply:
Sounds good :+1: